test/utility/misc.c

```diff
@@ -34,6 +34,12 @@
 void (*custom_die)(void) = NULL;
 const char *progname = "program name not set";
 
+/*
+ * Backup directory initialize command, remove and re-create the primary backup directory, plus a
+ * copy we maintain for recovery testing.
+ */
+#define HOME_BACKUP_INIT_CMD "rm -rf %s/BACKUP %s/BACKUP.copy && mkdir %s/BACKUP %s/BACKUP.copy"
```

Comment:
Not sure if wtperf runs only on linux machines here. Otherwise it wouldn't be compatible if wtperf was ran on other systems.

Reply:
I'm following the implementation of test/format. I think currently it is not running in windows.